### PR TITLE
Market data brühl

### PR DESCRIPTION
**Pull request by zliw**

### CODE REVIEW

File: cities/brühl.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                "coordinates": [
+                    6.90468,
+                    50.82831
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Frischemarkt Innenstadt",
+                "location": "Uhlstraße",
+                "opening_hours": "Mi 08:00-19:00"
```
Comment (johnjohndoe) [inline cities/brühl.json]: Looks like our validation script does not reliable - it detected an error but did not fail.

> BRÜHL: Frischemarkt Innenstadt
> Error: Mi <--- (Bitte benutze die englische Abkürzung "We" für "mi".)
> Validation done. 0 warning(s), 1 error(s) detected.

Can you please fix the weekday?

File: cities/brühl.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            "type": "Feature"
+        }
+    ],
+    "metadata": {
+        "data_source": {
+            "title": "Stadt Brühl - Wochenmärkte",
+            "url": "http://www.bruehl.de/wirtschaft/wirtschaftsfoerderung/wochenmaerkte.php"
+        },
+        "map_initialization": {
+            "coordinates": [
```
Comment (johnjohndoe) [inline cities/brühl.json]: bruhl

The center point could be a bit lower for nice rendering on small screens.

**Comment by zliw** [comment]
> The center point could be a bit lower for nice rendering on small screens.

I'm not sure, what the intention of this request is. Do you want :
1. to see all markets on small screens 
2. or do you want to have the main market visible on small screens
3. to have the exact center of the city visible on small screens

I assumed number 1 (and moved the center of the map a little bit north)

**Comment by johnjohndoe** [comment]
@zliw I squashed your commits into a single commit and merged the resulting branch here: 7569ced5d6be2260efd3d38bbace5488664b9b64. This means:

Brühl, welcome to _Wo ist Markt?_ :rocket: Thank you for your contribution :heart_eyes_cat:  

In order to promote the project I would like to ask you for two additional tasks:
- [ ] Send a tweet `#woistmarkt` with a **screenshot** and a **link** to the website (please share the tweet link here)
- [ ] Try to convince the city of Brühl to **link** to our website, see #38 

You are also very welcome to improve the **code**. Please take a look at our issue tracker.

**Pull request closed without merging**